File: Practica_8/src/Practica/Python/Main.py
from D import D

def main():
    d = D(1, 2, 3)

    d.incrementaXYZ()
    d.incrementaXZ()
    d.incrementaYZ()
    #Mostramos
    print(f"x: {d.x}, y: {d.y}, z: {d.z}")

#Ejecucion del Programa
if __name__ == "__main__":
    main()

File: Practica_8/src/Practica/Main.java
package Practica;

public class Main {

	public static void main(String[] args) {
		D d = new D(1, 2, 3);
		// Incrementa x, y, z
        d.incrementaXYZ();
        // Incrementa x y z de A
        d.a.incrementaXZ(); 
        // Incrementa y y z de B
        d.b.incrementaYZ();
        // Incrementa z de A
        d.a.incrementaZ();  
        // Incrementa z de B
        d.b.incrementaZ(); 

        d.mostrar();
	}
}
